hack/vendor-console.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
```

Comment:
we prefer
```sh
set -o errexit 
set -o nounset 
set -o pipefail
```

as these are easier to look at than the single-letter stuff and are a superset of `set -e`